examples/elm-pb/Python/data/BOUT.inp

```diff
@@ -38,8 +38,6 @@ group_nonblock = false  # Use non-blocking group operations?
 ##################################################
 # Laplacian inversion routines
 
-[laplace]
-
 # max_mode = 16  # Maximum N to solve for
```

Comment:
Same 'bug' as previous comment, but this should definitely not happen - orphaned commented-out option.